packages/material-ui-based-components/src/components/TabsHorizontalComp.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useContext } from 'react';
+import { getRenderer, FormContext } from '@aemforms/af-react-renderer';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+// import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { PROPS_PANEL } from '../utils/types';
+import { withRuleEnginePanel } from '../shared/withRuleEngine';
+
+interface TabPanelProps {
```

Comment:
please create the interface in type.ts file

Reply:
Made an interface in type.ts